src/pages/staticPages/tools/oncoprinter/Oncoprinter.tsx

```diff
@@ -171,9 +171,10 @@ export default class Oncoprinter extends React.Component<IOncoprinterProps, {}>
             onClickDownload:(type:string)=>{
                 switch(type) {
                     case "pdf":
-                        if (!svgToPdfDownload("oncoprint.pdf", this.oncoprint.toSVG(true))) {
-                            alert("Oncoprint too big to download as PDF - please download as SVG.");
-                        }
+                        svgToPdfDownload("oncoprint.pdf", this.oncoprint.toSVG(false));
```

Comment:
maybe wrap this in try catch and notify user if there's an error

Reply:
Sure I can do that. But should we have a limitation on the file size of SVG? For a very large oncoprint SVG, it may take half a minute for downloading.